### ISSUE TITLE
Better Rolls is not functioning with the newest update to the DnD 5e system.

### ISSUE THREAD

**Issue by TatianaKat** [RedReign/FoundryVTT-BetterRolls5e#96]
Hi there,

I'm not sure if this is a conflict with another module but the latest update to the 5e system that dropped.. today, I think, seems to have broken Better Rolls for me. It still pops up in the config settings but I'm no longer seeing it's tab on any feature details, nor is the roll functionality working.

**Comment by pdf** [comment]
This line imports a file that no longer exists, likely the source of the issue:

https://github.com/RedReign/FoundryVTT-BetterRolls5e/blob/92d49c54b5abe3a5e70b597a406c9f0a4701d510/betterrolls5e/scripts/betterrolls5e.js#L2

**Comment by DarKDinDoN** [comment]
Yep, Spell Dialod was revamped with the newest version of DND5e.

```
- Eliminate the SpellCastDialog class by expanding the flexibility of the AbilityUseDialog to encompass all types of item usages with both limited uses as well as spell slot consumption.
- The updated AbilityUseDialog can now handle cases of spell items which also have limited uses - for example an innate spell which can only be cast a certain number of times per day.
```

I think the author needs to update as well, so we don't lose features with BetterRolls (limited uses & slot consumption).

**Comment by RedReign** [comment]
Fixed in 1.1.10! Thank you all for the report.
Friendly reminder to not update to 1.1.10 if you are not on dnd5e 0.95+
